### ISSUE TITLE
Embed CSS by default (i.e., linkcss!)

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#428]
After having a chance to observe users' experience with the default stylesheet, I've determined that linking to, rather than embedding, the default stylesheet by default is not optimal. Many people have yet to discover the default stylesheet because they don't know either the `linkcss!` or `copycss` are required. If we embed by default, then it "works out of the box" and there is room for the user to discover that they can link to the stylesheet instead.

If we reverse this default, then it makes sense to copy the stylesheet by default when `linkcss!` is declared, because clearly the user expects the stylesheet to be present in the output folder.

In summary, this issue request proposes two changes:
- change the default setting linkcss to disabled (e.g., `linkcss!`)
- copy the default stylesheet to the output folder if linkcss is set (e.g., `linkcss`)
